save-deploy/README.md

```diff
@@ -23,6 +23,7 @@ Deployment is performed on server via docker swarm or locally via docker-compose
 ## Ports allocation
 | port | descritpion |
 | ---- | ----------- |
+} 3306 | database (locally) |
```

Comment:
Guess that we need to delete h2 console here, since we are not using it

Reply:
right, thx